### ISSUE TITLE
Removing links from case is not working

### ISSUE THREAD

**Issue by campgurus** [EBWiki/EBWiki#3612]
In Edit form, you can click "remove link" but the change is not saved in the DB, s.t. refreshing the page shows the deleted link again.

**Comment by thalees** [comment]
Hey, I can't login or create a local user with access to check this edit form. Is there a doc file that I can look or any instructions that can help me?

**Comment by rlgreen91** [comment]
@thalees Yes, if you look in the README on [the project page here](https://github.com/EBWiki/EBWiki#development), there's a link to two guides: one for setting up using Docker, and one for setting everything up locally.  Let us know if you run into any trouble with setup!
